test/controllers/api/docs_controller_test.exs

```diff
@@ -13,10 +13,10 @@ defmodule HexWeb.API.DocsControllerTest do
 
   @tag :integration
   test "release docs" do
-    user           = HexWeb.Repo.get_by!(User, username: "eric")
-    {:ok, phoenix} = Package.create(user, pkg_meta(%{name: "phoenix", description: "Web framework"}))
-    {:ok, _}       = Release.create(phoenix, rel_meta(%{version: "0.0.1", app: "phoenix"}), "")
-    {:ok, _}       = Release.create(phoenix, rel_meta(%{version: "0.0.2", app: "phoenix"}), "")
+    user = HexWeb.Repo.get_by!(User, username: "eric")
```

Comment:
I agree, there are lots of improvements that can be done to the testing. It has a lot of legacy debt from moving between different ecto versions and moving to phoenix.